### ISSUE TITLE
Is Book Text Screen Reader Friendly For Blind Users?

### ISSUE THREAD

**Issue by BobCochran** [karlseguin/the-little-mongodb-book#39]
Hi, is the pdf book text screen reader friendly -- that is, can screen reading software such as JAWS convert the text of the book and the code examples into speech for a blind user? I am not blind myself, but I work with two individuals who are. I'm not sure how to get started, but I imagine one way is to ask a blind user to try and work with the PDF version of this book. If readability of the PDF has never been verified by blind users before, can you please let me know? I will then talk to my associates, and ask if they are willing to put the book through their screen reading software. If they report that it does not work, I will ask if they are willing to advise me in how to fix the issues with the screen readers, by creating a new presentation format for the book. If at least one of these people agree, I will consider forking this project and working to make a book format that is screen reader friendly. 

To summarize I am asking if a determination has been made about whether the presentation formats of this book are usable by screen reading software used by blind individuals. If the answer is no, I am happy to ask some blind associates whom I work with whether they are willing to test the "screen read-ability" of the book in their screen reading equipment. From there, I will determine whether a new fork is advisable or possible. Thank you!
